common/dev/adm1272.c

```diff
@@ -253,22 +253,21 @@ int adm1272_read_iin(uint8_t sensor_num, float *val)
 	return 0;
 }
 
-uint8_t adm1272_read(uint8_t sensor_num, int *reading)
+uint8_t adm1272_read(void *arg, int *reading)
```

Comment:
Can we avoid using `void *` throughout this?  It seems like in most cases you've passed it along as a `sensor_cfg *` type.  Why is this one and some of the others different?  It seems like it would be ideal if we had no new `void *` added.